exonum-java-binding-time-service/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.exonum.binding</groupId>
+        <artifactId>exonum-java-binding-parent</artifactId>
+        <version>0.4-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>exonum-java-binding-time-service</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Exonum Time Service</name>
+    <url>https://exonum.com/doc/version/latest/advanced/time/</url>
+
+    <properties>
+        <ejb-core.nativeLibPath>${project.parent.basedir}/exonum-java-binding-core/rust/target/debug
+        </ejb-core.nativeLibPath>
+        <junit.jupiter.version>5.3.2</junit.jupiter.version>
+    </properties>
+
+    <dependencyManagement>
```

Comment:
Why do we need 'depManagement' and EJB-BOM? Aren't all dependencies already listed in 'depManagement' in the *parent*?